app/views/content_items/service_manual_guide.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= content_for :page_class, "service-manual" %>
+<%= content_for :page_class, "u-margin-bottom-large" %>
```

Comment:
Are we getting much benefit from using a `content_for` for this? The only thing that isn't in this template (at the moment) is the phase label, which I don't think needs to be inside `.u-margin-bottom-large`?

I thought the intent of this thing was a kind of convention to allow us to apply custom styles to a particular page type.

Could we bin it?